pkg/opssight/ctl_opssight.go

```diff
@@ -45,57 +46,31 @@ import (
 // generate the spec by telling CRSpecBuilderFromCobraFlags what flags were changed.
 type CRSpecBuilderFromCobraFlags struct {
 	opsSightSpec                                    *opssightapi.OpsSightSpec
-	PerceptorName                                   string
-	PerceptorImage                                  string
-	PerceptorPort                                   int
+	IsUpstream                                      string
```

Comment:
This seems to be used strictly for the component names and not the image locations.  What's the intent of this option?

Reply:
both images and names